cartopy/documentation.html

```diff
@@ -53,14 +54,32 @@
           <h2>Previous releases</h2>
             <ul>
                 <li><a href="docs/v0.13/index.html">v0.13</a></li>
+            </ul>
```

Comment:
Well, you say you inherited it which is fine, but you should note that this file is the one that actually appears online and it does not have these extra `<ul>` elements before this PR, you can check this by viewing the source of the live docs page. Therefore it would be *your* change that introduces these to the live docs.